syntax = "proto3";

package archiverestoreservice;

service ArchiveRestoreService {
  rpc ArchiveFunctionVersion (FunctionRequest) returns (FunctionResponse);
  rpc RestoreArchivedVersion (FunctionRequest) returns (FunctionResponse);
}

message FunctionRequest {
  string function_name = 1;
  string version = 2;
}

message FunctionResponse {
  bool success = 1;
  string message = 2;
}

